tests/integration-tests/conftest.py

```diff
@@ -422,11 +422,44 @@ def vpc_stacks(cfn_stacks_factory, request):
     """Create VPC used by integ tests in all configured regions."""
     regions = request.config.getoption("regions")
     vpc_stacks = {}
+
     for region in regions:
         # Randomly select 2 AZs from list WITHOUT replacement, hence the need for [None, None].
         # Creating private_subnet_different_cidr in a different AZ for test_efs
         # To-do: isolate this logic and create a compute subnet in different AZ than master in test_efs
-        availability_zones = random.sample(AVAILABILITY_ZONE_OVERRIDES.get(region, [None, None]), k=2)
+
+        # if region in AVAILABILITY_ZONE_OVERRIDES keys, find the availability_zones
+        if region in AVAILABILITY_ZONE_OVERRIDES:
+            availability_zones = random.sample(AVAILABILITY_ZONE_OVERRIDES.get(region), k=2)
+        # else if region is not in AVAILABILITY_ZONE_OVERRIDES keys, find available zones mapping to the region
+        else:
+            # get ec2 client
+            client = boto3.client('ec2', region_name=region)
+            response_az = client.describe_availability_zones(
```

Comment:
Minor: It would be good if we can batch the boto3 call and get all the AZ with one call of `describe_availability_zones`. However I just checked and this is not possible. `describe_availability_zones` only returns AZs in the client's region.

So implementation here is good, no modification needed